cmd/deploy/deploy.go

```diff
@@ -407,8 +410,8 @@ func (dc *DeployCommand) RunDeploy(ctx context.Context, deployOptions *Options)
 		data.Status = pipeline.DeployedStatus
 	}
 
-	if err := dc.CfgMapHandler.updateConfigMap(ctx, cfg, data, err); err != nil {
-		return err
+	if errStatus := dc.CfgMapHandler.updateConfigMap(ctx, cfg, data, err); errStatus != nil {
```

Comment:
is this rename for better reading? `err` is local variable of the if statement, so this changes just the name of the err